apps/src/cookieBanner/cookieBanner.js

```diff
@@ -0,0 +1,27 @@
+import cookies from 'js-cookie';
+import { getRootDomainFromHostname } from '@cdo/apps/code-studio/utils';
+
+window.setupCookieBanner = (environment) => {
+  const cookieName = '_cookieBanner' +
+    (environment === 'production' ? '' : ('_' + environment));
+  const banner = document.getElementById("cookie-banner");
+  const bannerButton = document.getElementById("accept-cookies");
+  const rootDomain = getRootDomainFromHostname(document.location.hostname);
+  const value = cookies.get(cookieName);
+
+  // Only show the cookie banner on test environment if there is a special
+  // URL parameter, which will be used for UI testing.
+  // Also temporarily hide it on production while we test internally.
+  const hideCookie =
```

Comment:
Variable naming: `hideCookieBanner` and maybe pull the cookie check up to here too.